docs/reference/filters.md

```diff
@@ -450,6 +450,21 @@ The second filter will set `Authorization` header to the
 `access_token` query param with a prefix value `Bearer ` and will
 not override the value if the header exists already.
 
+## TLS
+
+Filters that provide access to TLS data of a request.
+
+### tlsPassClient
+
+This filter copies TLS client certificate encoded as pem into the
```

Comment:
I think it could be multiple certificates, no?

Reply:
yes true

Reply:
done